python/servo/package_commands.py

```diff
@@ -674,6 +677,22 @@ def upload_to_s3(platform, package, timestamp):
                 }
             )
 
+            if github_release_id:
+                from github import Github
+                g = Github(os.environ['NIGHTLY_REPO_TOKEN'])
+                nightly_repo = g.get_repo(os.environ['NIGHTLY_REPO'])
+                release = nightly_repo.get_release(int(github_release_id))
+                if '2020' in platform:
+                    asset_name = f'servo-latest-layout2020.{extension}'
```

Comment:
Nit: `layout-2020` for the sake of consistency and I would use `nightly` instead of `latest`.

Reply:
Changed `layout2020` to `layout-2020` in 61fb795. 

As discussed over Matrix, I've retained the 'latest' suffix for 2020 to be consistent with existing naming convention for nightly builds, but this can be changed in the future if needed since only our internal WPT dashboard uses 2020 builds today.